js/components/rubric-box.js

```diff
@@ -30,13 +31,12 @@ export default class RubricBox extends PureComponent {
     const critId = crit.id
     const ratingId = rating.id
     const radioButtonKey = `${critId}-${ratingId}`
-    const checked = rubricFeedback &&
-      rubricFeedback[critId] &&
-      rubricFeedback[critId].id === ratingId
-    const ratingDescription = (
-      crit.ratingDescriptions &&
-      crit.ratingDescriptions[ratingId]
-    ) || null
+    const checked = !!(rubricFeedback && rubricFeedback[critId] && rubricFeedback[critId].id === ratingId)
+    // Tooltips displayed to teacher should actually show student description if it's available.
+    const ratingDescription =
+      (crit.ratingDescriptionsForStudent && crit.ratingDescriptionsForStudent[ratingId]) ||
+      (crit.ratingDescriptions && crit.ratingDescriptions[ratingId]) ||
+      null
```

Comment:
Couldn't we use the rubric helper here?

Reply:
Not really, as I think it's student-oriented. It's not used in this component at all. It helps with student box (to connect all the ratings with descriptions and so on), but not sure if it's that useful here. If we wanted to extend it, maybe we should just create set of independent functions instead of monolith class that handles all the cases.

If I remember correctly, `RubricHelper` accepts rubric JSON and particular rubric feedback that we don't have here (as we're just rendering table to let teacher create it).